Lib/pstats.py

```diff
@@ -333,6 +333,45 @@ def eval_print_amount(self, sel, list, msg):
 
         return new_list, msg
 
+    def get_profile_dict(self, keys_filter=None):
+        """
+        Returns a dict where the key is a function name and the value is a
+        dict with the following keys:
```

Comment:
Is returning a dict with keys ideal?  That feels old style vs returning an object with attributes.  A dataclass perhaps?

Reply:
related to this, is there a good reason to have the keys_filter feature?  (it wouldn't make as much sense on a dataclass, i guess filtered fields could be None?)

Reply:
Thank you for the suggestion! I removed remove `keys_filter` to keep things clean and changed to using `dataclass`, which I didn't know about before but it's awesome.